governance/paper-process.md

```diff
@@ -0,0 +1,169 @@
+# Process for creating papers
+
+This document is intended to provide a consistent mechanism for cloud native
+security to produce community papers, ensure they are reviewed, and subsequently
+published.  Content of this document does not supercede existing processes and
+is intended to be used in conjuction with [existing proposal to project
+instructions](process.md).
+
+## Proposal
+
+If a proposal is made that includes a paper as a deliverable, the proposal needs
+to ensure that there is a clearly idenitified lead and a well defined paper
+scope.
+
+The paper scope and topic should be raised in at least one SIG meeting to
+solicit more volunteers (ideally 4).  Interested parties should meet at least
+once to describe the intent of the paper, & propose a very rough outline to
+present to SIG leadership for planning and scheduling as a project.
+
+## Project
+
+Once a SIG Leadership sponsor is assigned to the project, the group should meet
+to and agree on a tentative schedule.
+
+### Tentative schedule milestones
```

Comment:
Is the below timeline gathered from the experience while creating the Cloud Native Security Whitepaper ?

Reply:
It was an amalgamation of the CNSWP as well as the supply chain.  It is here as a way to set expectations on the minimal time to create/write and publish a paper